precompiles/avs/avs.sol

```diff
@@ -14,11 +14,20 @@ IAVSManager constant AVSMANAGER_CONTRACT = IAVSManager(
 /// @custom:address 0x0000000000000000000000000000000000000902
 interface IAVSManager {
     function AVSAction(
```

Comment:
Why not name it RegisterAVS or InitializeAVS? AVSAction doesn't clearly indicate what the interface is doing.